frontend/src/features/search/SearchResultItem.tsx

```diff
@@ -1,57 +1,70 @@
 import React from 'react'
 import numeral from 'numeral'
 import { Link } from 'react-router-dom'
+import ContentLoader from 'react-content-loader'
 
 import DatasetInfoItem from '../dataset/DatasetInfoItem'
 import RelativeTimestamp from '../../chrome/RelativeTimestamp'
 
 import { SearchResult } from '../../qri/search'
 
 interface SearchResultItemProps {
-  dataset: SearchResult
+  dataset?: SearchResult
+  loading?: boolean
 }
 
-const SearchResultItem: React.FC<SearchResultItemProps> = ({ dataset }) => {
-    const {
-      peername,
-      name,
-      meta,
-      commit,
-      structure,
-      stats,
-      followStats
-    } = dataset
-    const datasetReference = `${peername}/${name}`
-
-    const title = meta?.title || datasetReference
-
-    const description = meta?.description
-
-    const timestamp = new Date(commit.timestamp)
-
-    const userIcon = (
-      <div className='rounded-xl inline-block mr-1 bg-cover flex-shrink-0' style={{
-        height: '18px',
-        width: '18px',
-        backgroundImage: 'url(https://qri-user-images.storage.googleapis.com/1570029763701.png)'
-      }} />
-    )
+const SearchResultItem: React.FC<SearchResultItemProps> = ({ dataset, loading=false }) => {
+    let itemContent
+
+    if (loading) {
+      itemContent = (
+        <ContentLoader viewBox="0 0 780 126">
+          <rect width="448" height="22" rx="6"/>
+          <rect y="30" width="401" height="29" rx="6"/>
+          <rect y="73" width="674" height="17" rx="6"/>
+          <rect y="101" width="420" height="25" rx="6"/>
+        </ContentLoader>
+      )
+    } else if (dataset) {
+      const {
+        peername,
+        name,
+        meta,
+        commit,
+        structure,
+        stats,
+        followStats
+      } = dataset
+      const datasetReference = `${peername}/${name}`
+
+      const title = meta?.title || datasetReference
+
+      const description = meta?.description
+
+      const timestamp = new Date(commit.timestamp)
+
```

Comment:
remove extraneous newlines, description variable doesn't seem like it needs to exist